src/Controller/HomeController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\Routing\Annotation\Route;
+
+class HomeController
+{
+    /**
+     * @Route ("/", name="home")
+     * @Template("index.html.twig")
```

Comment:
Template name is not needed. The template should be in the appropriate directory to autoload